### PR TITLE
Exclude iOS browser from tests related to negative bloom filter and subrequests

### PR DESCRIPTION
**Pull request by jaceklyp**

### CODE REVIEW

File: https-upgrades/tests.json [hunk 1]
```diff
@@ -42,7 +42,8 @@
                 "requestType": "main_frame",
                 "expectURL": "http://negativetest.com/path/file.html?search=query&another=one#fragment",
                 "exceptPlatforms": [
-                    "android-browser"
+                    "android-browser",
+                    "ios-browser"
```
Comment (kdzwinel) [inline https-upgrades/tests.json:46]: we have a separate tag for macos ("macos-browser") - https://github.com/duckduckgo/privacy-reference-tests/blob/main/CONTRIBUTING.md#exceptplatforms . Are BSK tests checking only "ios-browser" tag? Or are they checking for both? Or are those tests actually in iOS?

Comment (jaceklyp) [reply https-upgrades/tests.json:46]: I'm aware of that, these tests are implemented inside the BSK, so they test the common code for both platforms. Ideally we should have apple-browsers or apple-platforms, but Bartek told me that for now we should just check against ios-browser inside the BSK (I believe we do it in couple of tests already). However, I'm open to any suggestions here.

Comment (kdzwinel) [reply https-upgrades/tests.json:46]: all good, just wanted to understand how you are using those tags 👍

**Review by kdzwinel — APPROVED** [review]
One last thing - please document those exceptions in the readme - https://github.com/duckduckgo/privacy-reference-tests/tree/main/https-upgrades#platform-exceptions and feel free to merge afterwards.

Thank you!

**Comment by kdzwinel** [comment]
Perfect, thank you!

**Pull request merged**